### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by Raaghavi**

### CODE REVIEW

File: lib/fluent/plugin/filter_imds.rb [hunk 1]
```diff
@@ -82,11 +82,11 @@ def filter(tag, time, record)
         record["subscriptionId"] = data["compute"]["subscriptionId"]
         record["region"] = data["compute"]["location"]
         record["resourceGroupName"] = data["compute"]["resourceGroupName"]
-        record["resourceId"] = data["compute"]["resourceId"]
+        record["azureResourceId"] = data["compute"]["resourceId"]
```
Comment (boumenot) [inline lib/fluent/plugin/filter_imds.rb:85]: I understand the other changes, but I don't know why we added the `azure` prefix for resourceId.

Comment (Raaghavi) [reply lib/fluent/plugin/filter_imds.rb]: Reverted back to resourceId as discussed offline.

Comment (boumenot) [reply lib/fluent/plugin/filter_imds.rb]: We decided to delete resourceId, correct?

Comment (Raaghavi) [reply lib/fluent/plugin/filter_imds.rb]: Removed :)

**Pull request merged**